projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1645,6 +1651,21 @@ public function process_submission() {
 		wp_redirect( $redirect );
 		exit( 0 );
 	}
+	/**
+	 * Get the permalink for the post ID that include the page query parameter if it was set.
+	 *
+	 * @param int $post_id The post ID.
+	 *
+	 * return string The permalink for the post ID.
+	 */
+	public static function get_permalink( $post_id ) {
+		$url  = get_permalink( $post_id );
+		$page = isset( $_POST['page'] ) ? sanitize_text_field( wp_unslash( $_POST['page'] ) ) : null; // phpcs:Ignore WordPress.Security.NonceVerification.Missing
+		if ( $page ) {
+			return add_query_arg( 'page', $page, $url );
+		}
+		return $url;
+	}
```

Comment:
Would absint() be better to sanitize $_POST['page'] if we assume its always a positive integer?

Reply:
Good call!

Reply:
Not sure what convention is - I often like avoiding creating method names that exactly match built in WP methods. If that's a concern, could update this. Not a biggie though - it's already in a class and is always called with that class.